README.md

```diff
@@ -82,6 +82,7 @@ raccoon.recommendFor('chrisId', 10, function(results){
 raccoon.config.nearestNeighbors = 5;  // number of neighbors you want to compare a user against
 raccoon.config.className = 'movie';  // prefix for your items (used for redis)
 raccoon.config.numOfRecsStore = 30;  // number of recommendations to store per user
+raccoon.config.updateRatingOnInput = true;  // update recommendations on each input
```

Comment:
Thanks for the PR! Do you think it makes sense to put this in config if you're going to be toggling it multiple times during it's use. I think it belongs more as an optional parameter.